addons/rose/addon/styles/rose/components/form/_checkbox.scss

```diff
@@ -44,8 +44,6 @@
       height: sizing.rems(m);
       pointer-events: none;
       content: ' ';
-      border: sizing.rems(xxxxs) solid var(--ui-gray-subtler-2);
```

Comment:
Make no changes to the base checkbox.

Reply:
removed this. will update `outline` logic

Reply:
The diff is showing that these lines are still removed.  There should be no changes to the base checkbox styles.